Cargo.lock

```diff
@@ -254,7 +254,7 @@ checksum = "e78d4f1cc4ae33bbfc157ed5d5a5ef3bc29227303d595861deb238fcec4e9457"
 
 [[package]]
 name = "esrs"
-version = "0.6.0"
+version = "0.6.1"
```

Comment:
I think we need to gitignore Cargo.lock since this is a library.

Reply:
👍🏻 agree